### ISSUE TITLE
Some weights of the model checkpoint at ./model_hub/chinese-bert-wwm-ext/ were not used when initializing BertM

### ISSUE THREAD

**Issue by yukePeng0815** [taishan1994/BERT-Relation-Extraction#17]
您好，运行训练代码时候会出现如下警告这是正常的吗？
Some weights of the model checkpoint at ./model_hub/chinese-bert-wwm-ext/ were not used when initializing BertModel: ['cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.dense.weight', 'cls.predictions.bias', 'cls.seq_relationship.bias', 'cls.predictions.decoder.weight', 'cls.seq_relationship.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.dense.bias']
- This IS expected if you are initializing BertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
C:\Users\whut\.conda\envs\threeExtraction\lib\site-packages\torch\nn\modules\rnn.py:82: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.1 and num_layers=1
  warnings.warn("dropout option adds dropout after all but last "
C:\Users\whut\.conda\envs\threeExtraction\lib\site-packages\transformers\optimization.py:391: FutureWarning: This implementation of AdamW is deprecated and will be removed in a future version. Use the PyTorch implementation torch.optim.AdamW instead, or set `no_deprecation_warning=True` to disable this warning
  warnings.warn(
![Uploading image.png…]()

**Comment by yukePeng0815** [comment]
![@9T@ Y2Z(TJYF8A_R%QDYOW](https://github.com/user-attachments/assets/ae242681-0080-4c14-84ac-0af5301c945c)

**Comment by taishan1994** [comment]
能正常训练就没有影响
